plugins/catalog-graphql/scripts/mock-data.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+echo "use this script to load your service with some mock data if needed!"
```

Comment:
remove file?

Reply:
yep can do, it's just from the backend boilerplate that I was experimenting with for the `create-backend-plugin` stuff I was experimenting with